windows/flutter/generated_plugin_registrant.h

```diff
@@ -0,0 +1,13 @@
+//
+//  Generated file. Do not edit.
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase